### PR TITLE
Archetypes, Relic Fragments and Helmets

### PR DESCRIPTION
**Pull request by ghost**
Introduced in the `src/items` folder
* `Archetypes.json` with _12_ archetypes info
* `Helmets.json` with _27_ helmets info
* `RelicFragments.json` _39_ relic fragments info 

**Archetypes**
* A build can have _1_ or _2_ archetypes
* Each archetype will have _1_ out of _3_ possible skills equiped at a time

**Helmets**
* A build can have _1_ or no helmets
* In the search modal, we can display just the `name` and `description` for now
* Ignore other stats to display in tool for now

**Relic Fragments**
* Apart from the image, this has just 2 info to be displayed - `name` and `description` 

---

Added
* `Archetypes.json` with archetypes data
* `Helmets.json` with helmets data
* `RelicFragments.json` with relic fragments data

### CODE REVIEW

File: src/items/RelicFragments.json [hunk 1]
```diff
@@ -0,0 +1,353 @@
+[
+  {
+    "itemId": 0,
+    "itemName": "Mythic Ammo Pickups",
```
Comment (verzio-x) [inline src/items/RelicFragments.json:4]: item description missing for relic fragments

File: src/items/Archetypes.json [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+        "itemImageLinkFullPath": "https://remnant2.wiki.fextralife.com/file/Monster-Hunter-World/thumbnails/mhws.png",
+        "itemHref": "/Minion:+Reaver",
+        "itemFullHref": "https://remnant2.wiki.fextralife.com<div class=\"col-sm-4 col-xs-4\">\n<h5 align=\"center\"><a class=\"wiki_link wiki_tooltip\" href=\"/Minion:+Reaver\" title=\"Remnant 2 Minion: Reaver\"><img alt=\"mnion reaver summoner skill remnant2 wiki guide 75px\" class=\"lazyload\" data-src=\"/file/Remnant-2/mnion_reaver_summoner_skill_remnant2_wiki_guide_75px.png\" src=\"/file/Monster-Hunter-World/thumbnails/mhws.png\" style=\"width: 75px; height: 75px;\" title=\"Remnant 2 Minion: Reaver\"/><br/>Minion: Reaver<br/></a></h5>\n</div>"
+      }
+    ]
+  },
+  {
+    "itemId": 12,
+    "itemName": "Ritualist",
+    "itemDescription": "\nAOE, Status Effects,\u00a0Damage\n",
```
Comment (verzio-x) [inline src/items/Archetypes.json:412]: remove stray char in item description

File: src/items/Archetypes.json [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+        "skillId": 1,
+        "skillName": "Void Cloak",
+        "itemImageLinkPath": "/file/Monster-Hunter-World/thumbnails/mhws.png",
+        "itemImageLinkFullPath": "https://remnant2.wiki.fextralife.com/file/Monster-Hunter-World/thumbnails/mhws.png",
+        "itemHref": "/Void+Cloak",
+        "itemFullHref": "https://remnant2.wiki.fextralife.com<div class=\"col-sm-4 col-xs-4\">\n<h5 align=\"center\"><a class=\"wiki_link wiki_tooltip\" href=\"/Void+Cloak\" title=\"Remnant 2 Void Cloak\"><img alt=\"void cloak invader skill remnant2 wiki guide 75px\" class=\"lazyload\" data-src=\"/file/Remnant-2/void_cloak_invader_skill_remnant2_wiki_guide_75px.png\" src=\"/file/Monster-Hunter-World/thumbnails/mhws.png\" style=\"width: 75px; height: 75px;\" title=\"Remnant 2 Void Cloak\"/><br/>Void Cloak<br/></a></h5>\n</div>"
+      },
+      {
+        "skillId": 2,
+        "skillName": "Worn Hole",
```
Comment (verzio-x) [inline src/items/Archetypes.json:356]: typo, worm instead of worn

File: src/items/Archetypes.json [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+        "skillId": 1,
+        "skillName": "War Stomp",
+        "itemImageLinkPath": "/file/Monster-Hunter-World/thumbnails/mhws.png",
+        "itemImageLinkFullPath": "https://remnant2.wiki.fextralife.com/file/Monster-Hunter-World/thumbnails/mhws.png",
+        "itemHref": "/War+Stomp",
+        "itemFullHref": "https://remnant2.wiki.fextralife.com<div class=\"col-sm-4 col-xs-4\">\n<h5 align=\"center\"><a class=\"wiki_link wiki_tooltip\" href=\"/War+Stomp\" title=\"Remnant 2 War Stomp\"><img alt=\"war stomp skill remnant2 wiki guide 75px\" class=\"lazyload\" data-src=\"/file/Remnant-2/war_stomp_skill_remnant2_wiki_guide_75px.png\" src=\"/file/Monster-Hunter-World/thumbnails/mhws.png\" style=\"width: 75px; height: 75px;\" title=\"Remnant 2 War Stomp\"/><br/>War Stomp<br/></a></h5>\n</div>"
+      },
+      {
+        "skillId": 2,
+        "skillName": "Juggernaut",
```
Comment (verzio-x) [inline src/items/Archetypes.json:23]: add `itemType` for skills as well and call it `archetype_skill`

**Comment by verzio-x** [comment]
@PattamKarthi this is good to merge

**Review by vrsax — APPROVED** [review]

**Review by vrsax — APPROVED** [review]

**Pull request merged**